units/meson.build

```diff
@@ -662,6 +662,14 @@ units = [
           'file' : 'systemd-resolved.service.in',
           'conditions' : ['ENABLE_RESOLVE'],
         },
+        {
+          'file' : 'systemd-resolved-varlink.socket',
+          'conditions' : ['ENABLE_RESOLVE'],
+        },
+        {
+          'file' : 'systemd-resolved-monitor.socket',
+          'conditions' : ['ENABLE_RESOLVE'],
```

Comment:
resolved.service should pull these sockets in

Reply:
Not addressed?

Reply:
Added in the last push.

I also added:

```
Before=shutdown.target
Conflicts=shutdown.target
```

to the socket units, as my understanding is that this is generally a good idea for clean shutdowns when `DefaultDependencies=no`. Correct me if we don't want that here.